<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gui_1Class</class>
 <widget class="QMainWindow" name="gui_1Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OUSB Control</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_16">
    <item row="0" column="0">
     <widget class="QLCDNumber" name="lcdNumber">
      <property name="minimumSize">
       <size>
        <width>64</width>
        <height>23</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>64</width>
        <height>23</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Switches</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0" colspan="3">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string/>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="1" column="0">
            <widget class="QSlider" name="switch0">
             <property name="maximum">
              <number>1</number>
             </property>
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item row="1" column="7">
            <widget class="QSlider" name="switch7">
             <property name="maximum">
              <number>1</number>
             </property>
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item row="1" column="6">
            <widget class="QSlider" name="switch6">
             <property name="maximum">
              <number>1</number>
             </property>
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item row="1" column="5">
            <widget class="QSlider" name="switch5">
             <property name="maximum">
              <number>1</number>
             </property>
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSlider" name="switch1">
             <property name="maximum">
              <number>1</number>
             </property>
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item row="1" column="4">
            <widget class="QSlider" name="switch4">
             <property name="maximum">
              <number>1</number>
             </property>
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QSlider" name="switch2">
             <property name="maximum">
              <number>1</number>
             </property>
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QSlider" name="switch3">
             <property name="maximum">
              <number>1</number>
             </property>
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>1</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>2</string>
             </property>
            </widget>
           </item>
           <item row="2" column="3">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>3</string>
             </property>
            </widget>
           </item>
           <item row="2" column="4">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>4</string>
             </property>
            </widget>
           </item>
           <item row="2" column="5">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>5</string>
             </property>
            </widget>
           </item>
           <item row="2" column="6">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>6</string>
             </property>
            </widget>
           </item>
           <item row="2" column="7">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>7</string>
             </property>
            </widget>
           </item>
           <item row="0" column="9">
            <widget class="QLabel" name="label_10">
             <property name="minimumSize">
              <size>
               <width>21</width>
               <height>16</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>21</width>
               <height>16</height>
              </size>
             </property>
             <property name="text">
              <string>ON</string>
             </property>
            </widget>
           </item>
           <item row="2" column="9">
            <widget class="QLabel" name="label_11">
             <property name="minimumSize">
              <size>
               <width>31</width>
               <height>16</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>31</width>
               <height>16</height>
              </size>
             </property>
             <property name="text">
              <string>OFF</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLCDNumber" name="switchNum">
          <property name="minimumSize">
           <size>
            <width>64</width>
            <height>23</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>64</width>
            <height>23</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="sendToLed">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>Send To LED</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>LEDs</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLCDNumber" name="ledNum">
          <property name="minimumSize">
           <size>
            <width>64</width>
            <height>23</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>64</width>
            <height>23</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string/>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="1" column="7">
            <widget class="QPushButton" name="led7">
             <property name="text">
              <string>7</string>
             </property>
            </widget>
           </item>
           <item row="1" column="6">
            <widget class="QPushButton" name="led6">
             <property name="text">
              <string>6</string>
             </property>
            </widget>
           </item>
           <item row="1" column="5">
            <widget class="QPushButton" name="led5">
             <property name="text">
              <string>5</string>
             </property>
            </widget>
           </item>
           <item row="1" column="4">
            <widget class="QPushButton" name="led4">
             <property name="text">
              <string>4</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QPushButton" name="led3">
             <property name="text">
              <string>3</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="led2">
             <property name="text">
              <string>2</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="led0">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="led1">
             <property name="text">
              <string>1</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>LDR And Pot</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>LDR</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="0" column="0" rowspan="2">
            <widget class="QDial" name="ldrDial"/>
           </item>
           <item row="0" column="1">
            <widget class="QLCDNumber" name="ldrNum">
             <property name="minimumSize">
              <size>
               <width>64</width>
               <height>23</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>64</width>
               <height>23</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QProgressBar" name="ldrBar">
             <property name="value">
              <number>24</number>
             </property>
             <property name="textVisible">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>Pot</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0" rowspan="2">
            <widget class="QDial" name="potDial"/>
           </item>
           <item row="0" column="1">
            <widget class="QLCDNumber" name="potNum">
             <property name="minimumSize">
              <size>
               <width>64</width>
               <height>23</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>64</width>
               <height>23</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QProgressBar" name="potBar">
             <property name="value">
              <number>24</number>
             </property>
             <property name="textVisible">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Spare Ports</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox_5">
          <property name="title">
           <string>PA0</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_9">
           <item row="0" column="0">
            <widget class="QLCDNumber" name="pa0Num">
             <property name="minimumSize">
              <size>
               <width>64</width>
               <height>23</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>64</width>
               <height>23</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QProgressBar" name="pa0Bar">
             <property name="value">
              <number>24</number>
             </property>
             <property name="textVisible">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QGroupBox" name="groupBox_6">
          <property name="title">
           <string>PA1</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_10">
           <item row="0" column="0">
            <widget class="QLCDNumber" name="pa1Num">
             <property name="minimumSize">
              <size>
               <width>64</width>
               <height>23</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>64</width>
               <height>23</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QProgressBar" name="pa1Bar">
             <property name="value">
              <number>24</number>
             </property>
             <property name="textVisible">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QGroupBox" name="groupBox_7">
          <property name="title">
           <string>PA2</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_11">
           <item row="0" column="0">
            <widget class="QLCDNumber" name="pa2Num">
             <property name="minimumSize">
              <size>
               <width>64</width>
               <height>23</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>64</width>
               <height>23</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QProgressBar" name="pa2Bar">
             <property name="value">
              <number>24</number>
             </property>
             <property name="textVisible">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBox_8">
          <property name="title">
           <string>PA3</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_14">
           <item row="1" column="0" rowspan="2" colspan="2">
            <widget class="QProgressBar" name="pa3Bar">
             <property name="value">
              <number>24</number>
             </property>
             <property name="textVisible">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLCDNumber" name="pa3Num">
             <property name="minimumSize">
              <size>
               <width>64</width>
               <height>23</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>64</width>
               <height>23</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QGroupBox" name="groupBox_9">
          <property name="title">
           <string>PA4</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_13">
           <item row="1" column="0">
            <widget class="QProgressBar" name="pa4Bar">
             <property name="value">
              <number>24</number>
             </property>
             <property name="textVisible">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLCDNumber" name="pa4Num">
             <property name="minimumSize">
              <size>
               <width>64</width>
               <height>23</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>64</width>
               <height>23</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QGroupBox" name="groupBox_10">
          <property name="title">
           <string>PA7</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_12">
           <item row="1" column="0">
            <widget class="QProgressBar" name="pa7Bar">
             <property name="value">
              <number>24</number>
             </property>
             <property name="textVisible">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLCDNumber" name="pa7Num">
             <property name="minimumSize">
              <size>
               <width>64</width>
               <height>23</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>64</width>
               <height>23</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Pot Chart</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_15">
        <item row="0" column="1">
         <widget class="QTextBrowser" name="textBrowser"/>
        </item>
        <item row="5" column="1">
         <widget class="QLabel" name="label_13">
          <property name="minimumSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QLabel" name="label_12">
          <property name="minimumSize">
           <size>
            <width>31</width>
            <height>16</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>31</width>
            <height>16</height>
           </size>
          </property>
          <property name="text">
           <string>1023</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_14">
          <property name="minimumSize">
           <size>
            <width>31</width>
            <height>16</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>31</width>
            <height>16</height>
           </size>
          </property>
          <property name="text">
           <string>Time</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label">
      <property name="minimumSize">
       <size>
        <width>221</width>
        <height>31</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>221</width>
        <height>31</height>
       </size>
      </property>
      <property name="text">
       <string>Student Name: Hayden Warmington
Student Number: S3486075</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>810</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
